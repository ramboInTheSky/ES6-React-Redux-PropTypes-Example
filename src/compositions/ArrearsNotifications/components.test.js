import React from 'react';
import { shallow } from 'enzyme';
import Container from './components';

describe('ArrearsNotifications components', () => {
  it('should render default container correctly', () => {
    expect(shallow(<Container />)).toMatchSnapshot();
  });
});
