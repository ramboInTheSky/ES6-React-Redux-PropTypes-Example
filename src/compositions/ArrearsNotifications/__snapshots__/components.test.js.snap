// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrearsNotifications components should render default container correctly 1`] = `
<div
  className="sc-bdVaJa eMmrIk"
/>
`;
